spot_rl_experiments/spot_rl/envs/skill_manager.py

```diff
@@ -506,6 +509,16 @@ def place(self, place_target: str = None, ee_orientation_at_grasping: np.ndarray
                 is_local = True
         else:
             message = "No place target specified, estimating point through heuristic"
+            height = calculate_height(place_target)
```

Comment:
since we are using this block of code twice, we can make this into a function and call the function in both place and constrainedplace to avoid code duplication